cogs/covidcog.py

```diff
@@ -0,0 +1,65 @@
+import discord
+from discord.ext import commands
+import requests
+
+
+def embedCreator(data):
+    """
+        Creates the embedded message for the covid-19 stats
+    """
+    state = data["state"]
+    embedTitle = ":world_map: Covid Stats in " + state + " :world_map:"
+    firstField = {
+        "name": "Total " + state + " Cases",
+        "value": "Total Cases: {:,}".format(data["cases"]),
+        "inline": True
+    }
+    secondField = {
+        "name": "Total " + state + " Deaths",
+        "value": "Total Deaths: {:,}".format(data["deaths"]),
+        "inline": True
+    }
+    thirdField = {
+        "name": "Total " + state + " Tests",
+        "value": "Total Tests: {:,}".format(data["tests"]),
+        "inline": True
+    }
+    embed = discord.Embed(title=embedTitle, color=0x8D0000)
+    embed.description = "Statistics for Covid-19 in " + state + "\n"
+    embed.add_field(
+        name=firstField["name"], value=firstField["value"], inline=firstField["inline"])
+    embed.add_field(
+        name=secondField["name"], value=secondField["value"], inline=secondField["inline"])
+    embed.add_field(
+        name=thirdField["name"], value=thirdField["value"], inline=thirdField["inline"])
+    embed.set_footer(text="Data from corona.lmao.ninja")
+
+    return embed
+
+
+class CovidCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def covid(self, ctx):
+        """
+        Requests data from corona.lmao.ninja and parses it based on the users
```

Comment:
Try to be very clear about how to use the command in the command's docstring, as this becomes its help message.